tutorials/containers/flux-sched-container.rst

```diff
@@ -0,0 +1,232 @@
+.. _flux-sched-container:
+
+====================================
+Quick Start with Flux in a Container
+====================================
+
+Do you want to submit a job to Flux? Here's a short tutorial on how to do so via a container!
+The reason this works is because a Flux instance can be run anywhere...
+
+    | You can run it on a cluster... 🥜️
+    | You can run it alongside Lustre! 📁️
+    | You can run it on a share... 🤗️
+    | You can run it anywhere! 🏔️
+    | You can run it in a container... 📦️
+    | Using Flux? Total no-brainer! 🧠️
+
+Flux has a regularly updated Docker image on `Docker Hub <https://hub.docker.com/u/fluxrm>`_. 
+You will need to `install docker <https://docs.docker.com/engine/install/>`_ first.
+
+----------------------
+Starting the Container
+----------------------
+
+Run the container as follows:
+
+.. code-block:: bash
+
+  # This says "run an interactive terminal"
+  $ docker run -it fluxrm/flux-sched:latest
+
+At this point, even before starting an instance, we can inspect what is going on with Flux.
```

Comment:
The entrypoint for all the docker images is `/usr/bin/flux start /bin/bash` so a test instance is already started.

Maybe reword to say something like "By default, the docker container starts a test Flux instance of size 1"